src/forms/EditJobForm/index.jsx

```diff
@@ -61,8 +77,11 @@ export const EditJobForm = ({ job, token, coinSymbol }) => {
             id="title"
             label="Title"
             placeholder=""
-            value={fields.title}
-            onChange={handleInputChange}
+            defaultValue={localStorage.getItem('editjobTitle') ?? ''}
```

Comment:
100% there is a bug. Did you test it in your browser?
I've tried to edit an existing job and I've got empty fields in the form.

If think it would be better to replace `''` in ternary operator with `fields.title`, `fields.budget` and `fields.description` (regarding concrete input field), because there are stored original values from component props. WDYT? :-)

Reply:
Also I think it would be good to store in specific key inside localStorage, like `editjobTitle-${job.id}`. If you try to open first job to edit, make some changes in `title` and then open another job for edit as well – original content in the second job form will be replaced with content from the first job card.

Reply:
I missed this one! Fixed now, I think. Tested on firefox and chrome.

Reply:
> will be replaced with conten

Done! Suffixed with -${job.id}